samples/bluetooth/unicast_audio_client/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+
+# For LC3 the following configs are needed
+CONFIG_LIBLC3CODEC=y
+CONFIG_FPU=y
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_NEWLIB_LIBC=y
```

Comment:
Is this additional intentional?

I'm thinking that if we enable the LIBLC3CODEC for the nRF5340 by default, we are going towards just using the LIBLC3CODEC module generally for this sample, and thus it should be moved to the default `prj.conf`.

Reply:
> But it will not run on a CPU that do not have HW FPU support

Would it make sense for it to depend on `CONFIG_FPU` then? `CONFIG_FPU` depends on `CPU_HAS_FPU`, so if we depend on `CONFIG_FPU`, then we can ensure that it can only be enabled on devices with FPU. 

But it definitely makes sense that it doesn't run on all boards, but if we add some sort of dependency, then we could enable it by default on the boards that support it.

Reply:
I made it select FPU. I don't think it makes to run this codec on a CPU with FPU disabled even though it could.